twopence/run

```diff
@@ -1133,22 +1140,18 @@ def nfs_test_sysvinit():
 	running = False
 
 	journal.beginTest("setup: write simple exports file to server")
-	if not nfs_write_exports(server, my_exports):
+	if not nfs_write_exports(server, my_exports, reload_nfs=False):
 		journal.failure("Unable to initialize sysvinit tests")
 		return False
 
-	journal.beginTest("Check status of stopped NFS server")
-	st = server.run("rcnfsserver status")
-	if st.code != 3:
-		journal.info("rcnfsserver status returned exit code %d" % st.code)
-		journal.failure("rcnfsserver should have returned 3 [unused]")
+	# expect failure with retcode 3. journal.msg at end is optional
+	assert_fail_ret(server, "rcnfsserver status", 3 , "Check status of stopped NFS server")
```

Comment:
Hi Dario,

I'm not sure I understand what this change is about.

Besides, it looks ugly to me :-)

Regards,
Olaf

-- 
Nous sommes du soleil
We love when we play

> Am 03.06.2016 um 17:38 schrieb Dario Maiocchi <notifications@github.com>:
> 
> In twopence/run:
> 
> >  		journal.failure("Unable to initialize sysvinit tests")
> >  		return False
> >  
> > -	journal.beginTest("Check status of stopped NFS server")
> > -	st = server.run("rcnfsserver status")
> > -	if st.code != 3:
> > -		journal.info("rcnfsserver status returned exit code %d" % st.code)
> > -		journal.failure("rcnfsserver should have returned 3 [unused]")
> > +	# expect failure with retcode 3. journal.msg at end is optional
> > +	assert_fail_ret(server, "rcnfsserver status", 3 , "Check status of stopped NFS server")
> this come from susetest_api.assertions
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>